overlord/snapstate/storehelpers.go

```diff
@@ -201,6 +203,13 @@ var installSize = func(st *state.State, snaps []minimalInstallInfo, userID int)
 	return total, nil
 }
 
+func setActionValidationSets(action *store.SnapAction, valsets []string) {
```

Comment:
I wonder, could this be rewritten as a method of SnapAction? Like
```
func (a *store.SnapAction) SetValidationSets(valsets []string) { ... }
```

Reply:
The reason I didn't make it a method is that SnapAction is a plain struct with all fields public and we don't have any methods for it; this one is just a simple helper to avoid repetition, I don't think it makes sense as a method.

Reply:
As you wish, I don't feel strongly about this :-)